source/future.tex

```diff
@@ -1468,8 +1468,7 @@
 only if the type \tcode{T} is any of the following:
 \begin{itemize}
 \item a function type or a pointer to function type taking one argument of type \tcode{T1}
-% FIXME: Should this be R T0::f() ?
-\item a pointer to member function \tcode{R T0::f} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
+\item a pointer to member function \tcode{R T0::f()} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
```

Comment:
I think it makes sense with the fix. As an example of what the item is talking about, consider the member function:

    size_t string::size() const

(R=size_t, T0=string, f=size, cv=const)

The item now says that if T (reference_wrapper's parameter) is a pointer to this thing, then T1 (the resulting argument_type) is `cv T0*`, which is `const string*`.

So it is saying that `reference_wrapper<size_t(string::*)()const>::argument_type` is `const string*`, which is also what I get with libstdc++ and libc++.